Monika After Story/game/event-rules.rpy

```diff
@@ -15,6 +15,7 @@ init -1 python:
     EV_RULE_AFF_RANGE = "affection_range"
     EV_RULE_PRIORITY = "rule_priority"
     EV_RULE_PROBABILITY = "rule_probability"
+    EV_RULE_REPEAT_TD = "rule_repeat_td"

```

Comment:
```suggestion
    EV_RULE_RP_TIMEDELTA = "rp_timedelta"
```

The new rule is the same idea as the previous repeat rules except it uses timedelta as the increment. Should keep the naming scheme in the same way.